src/main/kotlin/com/openlattice/hazelcast/serializers/SecurablePrincipalListStreamSerializer.kt

```diff
@@ -22,15 +22,51 @@ package com.openlattice.hazelcast.serializers
 
 import com.hazelcast.nio.ObjectDataInput
 import com.hazelcast.nio.ObjectDataOutput
+import com.openlattice.authorization.AclKey
 import com.openlattice.authorization.SecurablePrincipal
 import com.openlattice.hazelcast.StreamSerializerTypeIds
+import com.openlattice.mapstores.TestDataFactory
 import com.openlattice.organizations.SecurablePrincipalList
 import org.springframework.stereotype.Component
+import java.util.*
+import kotlin.collections.ArrayList
+import kotlin.random.Random
 
 @Component
 class SecurablePrincipalListStreamSerializer
     : ListStreamSerializer<SecurablePrincipalList, SecurablePrincipal>(SecurablePrincipalList::class.java) {
 
+    companion object {
+        @JvmStatic
+        fun createSecurablePrincipalList(): SecurablePrincipalList {
```

Comment:
Any reason not to put this in `generateTestValue()` ?
Its only used for testing so I think it would be best for it not to be static/in a production class.
Alternatively, could move it to `TestDataFactory`

Reply:
I will move it there. It's used in other test classes, for example: `AddMembersToOrganizationAssemblyProcessorStreamSerializerTest`